### DISCUSSION TITLE
Is it possible to intercept proxy request to CURL?

### DISCUSSION THREAD

**Discussion by rajhlinux** [curl/curl#10842] [Q&A]
I have a web browser, when I click on a button, I want CURL to display the headers and HTML body response.

How can this be done?

By the way I am using libcurl (C language).

**Comment by dfandrich** [comment]
The easiest way is usually to use the "Copy as curl" option that many browsers offer in their developer tools window.

**Comment by rajhlinux** [comment]
Thanks for your reply. I Actually meant like a button the HTML DOM and that sends a request to the server, I have a proxy on the chrome extension and have CURL intercept the request and display the headers and HTML body response. Wondering how this could be done. I'm able to make requests using CURL, which is great but would like to intercept proxied requests.

**Reply by bagder** [reply]
> have CURL intercept the request

curl is a client, it cannot intercept any requests.
